File: public/phase-2-20160620022203/C/pony-sequence.c
// Name : Pony Sequence
#include <stdio.h>
int main() {
	// input:
	// n : number of elements in the sequences in a and b
	// a : sequence a
	int n, a[100], b[100]={1,1,1,1,1}, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `b`

	// output
	for(index = 0; index< n; index++) {
		printf("%d ",b[index]);
	}
	return 0;
}

File: public/phase-2-20160620022203/JAVA/pony-sequence.java
// Name : Pony Sequence
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : number of elements in the sequences in a and b
		// a : sequence a
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] a = new int[n];
		String[] a_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			a[index] = Integer.parseInt(a_elements[index]);


		// write your code here
		// store your results in `b`

		// output
		// Dummy Data
		intb[] = new int[] {1,1,1,1,1};

		System.out.println(join_string(Arrays.toString(b)));

	}
	public static String join_string(String inp) {
		return inp.replace("[","").replace("]","").replace(",","");
	}
}